Sources/Clocks/UnimplementedClock.swift

```diff
@@ -64,65 +64,60 @@
   /// failures. This will help us find the tests that should be updated to assert on the new behavior
   /// in the feature.
   @available(iOS 16, macOS 13, tvOS 16, watchOS 9, *)
-  public final class UnimplementedClock<Duration>: Clock, @unchecked Sendable
-  where
-    Duration: DurationProtocol,
-    Duration: Hashable
-  {
+  public struct UnimplementedClock<Duration: DurationProtocol & Hashable>: Clock {
     public struct Instant: InstantProtocol {
-      public var offset: Duration
```

Comment:
We're technically removing this surface area, but I think it shouldn't be relied on externally (we had a test asserting against it but that assertion, too, I think can go away).